common/source/docs/common-understanding-altitude.rst

```diff
@@ -0,0 +1,82 @@
+hoving.. _common-understanding-altitude:
+
+===================================
+Understanding ALTITUDE in ArduPilot
```

Comment:
Let's use lowercase "Altitude" instead of upper case

Reply:
fixed